configure.ac

```diff
@@ -298,7 +298,15 @@ AC_SUBST([WRAPPER_COMPILER_CC])
 AC_MSG_CHECKING([if -std=gnu11 works])
 
 AC_LANG_PUSH([C])
-CFLAGS="-pedantic-errors -std=gnu11"
+C_COMPILER_VERSION=$("$CC" --version 2> /dev/null)
+IS_ONEAPI_COMPILER=$(echo $C_COMPILER_VERSION | $GREP -c -E 'Intel\(R\) oneAPI')
```

Comment:
GNU docs suggest using `AC_PROG_EGREP` before $GREP -E, which doesn't seem to be checked before this in configure.ac.

Reply:
I might not have got the comment. Do we need to add this check then? I am not seeing it added.

Reply:
Oh right, this was a reminder that I did not add the check, but it seems to work without it.

I looked into this briefly and I see that somewhere earlier there is already a check:
`checking for grep that handles long lines and -e... /usr/bin/grep`

I don't know exactly where that check occurs and if we can count on it...

Reply:
Yes, I remember this also.. and it happens very early in the configure. I think we can go with it. If someone reports a bug, we can add it then.